documentation/tutorial.md

```diff
@@ -68,3 +68,145 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
 Model "admin/tutorial" is empty.
 ```
 
+
+## Deploying Charmed MongoDB:
+To deploy Charmed MongoDB, fetch the charm from [Charmhub](https://charmhub.io/mongodb?channel=dpe/edge) and deploy it to your model:
+```
+juju deploy mongodb --channel dpe/edge
+```
+
+Juju will now fetch Charmed MongoDB and begin deploying it to the LXD cloud. This process can take several minutes depending on how provisioned your machine is. You can track the progress by running:
+```
+watch -c juju status --color
+```
+
+This will show charmed MongoDB along with its current status and other helpful information.
+
+Wait until the application is ready, when it is ready `watch -c juju status --color` will show
+```
+Every 2.0s: juju status --color                                                                                         ip-172-31-11-104: Fri Dec  2 11:24:30 2022
+
+Model     Controller  Cloud/Region         Version  SLA          Timestamp
+tutorial  overlord    localhost/localhost  2.9.37   unsupported  11:24:30Z
+
+App      Version  Status  Scale  Charm    Channel   Rev  Exposed  Message
+mongodb           active      1  mongodb  dpe/edge   96  no
+
+Unit        Workload  Agent  Machine  Public address  Ports      Message
+mongodb/0*  active    idle   0        10.23.62.156    27017/tcp
+
+Machine  State    Address       Inst id        Series  AZ  Message
+0        started  10.23.62.156  juju-d35d30-0  focal       Running
+```
+
+## Accessing MongoDB:
+*Disclaimer: this part of the tutorial accesses MongoDB via the `admin` user. It is not recommended that you directly interface with the admin user in a production environment. In a production environment [create a separate user](https://www.mongodb.com/docs/manual/tutorial/create-users/) and connect to MongoDB with that user instead.*
+
+For this part of the Tutorial we will access MongoDB - via the MongoDB shell `mongosh`. You can read more about the MongoDB shell [here](https://www.mongodb.com/docs/mongodb-shell/).
+
+
+### Install the MongoDB shell:
+While MongoDB is installed within the LXD containers that the application is hosted on, it is not installed on your machine. This means that you will have to install the MongoDB shell yourself. Install `mongosh` by following the official [instructions on how to install the MongoDB shell](https://www.mongodb.com/docs/mongodb-shell/install/#std-label-mdb-shell-install).
```

Comment:
what about launching an lxd container con mongoDB instead? This way we don't mess up with the user environment

Reply:
hmm interesting point. If we didn't want to "mess up with the user environment" we could just `ssh mongodb/unit` and then access mongodb via mongosh there (since mongosh is already installed). But I think doing either of these abstracts things a little too much from the reader. 

But also I don't think installing mongosh really messes up the environment since the user is also installing other packages in this tutorial

Reply:
Yes. It is not something major that I feel very strong about. But let's see what the other reviewer (@zmraul ) thinks about this

Reply:
2c: In SQL-land there's actually dedicated charms for shells, but that's obviously a lot more work. If I was running this charm, I'd expect to need a local shell if there wasn't a dedicated shell charm for me to use.

Reply:
Uhm, we don't have that and of course we should definitely not implement that within the context of this task. Should we then rather add this in the requirement bit?

Reply:
I think installing `mongosh` on the host is ok here. My other choice would be to `ssh` into the unit to see it working as Mia said.

Reply:
okay cool lets keep the install of mongosh, thank you everyone